src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -56,4 +60,17 @@ public SendLetterController(
 
         return ok().body(messageId);
     }
+
+    @PutMapping(path = "/{id}/sent-to-print-at")
+    @ApiOperation(value = "Update when letter was sent to print")
+    public ResponseEntity<Void> updateSentToPrint(
+        @PathVariable("id") String id,
+        @RequestBody LetterSentToPrintAtPatch patch,
+        @RequestHeader("ServiceAuthorization") String serviceAuthHeader
+    ) {
```

Comment:
Using String ID for now, we'll see how db will react to invalid uuid in next PR.